src/components/Card.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components/native'
+import {metrics} from '../helpers/metrics'
+import type {ITheme} from '../theme'
+import type {ViewProps} from 'react-native'
+
+export type CardProps = {
+  onPress?: () => void
+  children?: React.ReactNode
+  bgColor?: string
+  activeOpacity?: number
```

Comment:
why define activeOpacity here but don't use in component?

Reply:
Updated it